qiskit/circuit/quantumcircuit.py

```diff
@@ -2273,6 +2279,29 @@ def cz(self, control_qubit, target_qubit, *, label=None, ctrl_state=None,
         return self.append(CZGate(label=label, ctrl_state=ctrl_state),
                            [control_qubit, target_qubit], [])
 
+    def add_calibration(self, gate, qubits, schedule, params=None):
+        """Register a low-level, custom pulse definition for the given gate.
+
+        The custom pulse definition of a given gate is of the form
+            {'gate_name': {(qubits, params): schedule}}
+
+        Args:
+            gate (Union[Gate, str]): Gate information.
+            qubits (Union[int, Tuple[int]]): List of qubits to be measured.
+            schedule (Schedule): Schedule information.
+            params (Optional[List[Union[float, Parameter]]]): A list of parameters.
+
+        Raises:
+            Exception: if the gate is of type string and params is None.
+        """
+        if isinstance(gate, Gate):
+            self._calibrations[gate.name][(tuple(qubits), tuple(gate.params))] = schedule
+        elif isinstance(gate, str) and params is not None:
+            self._calibrations[gate][(tuple(qubits), tuple(params))] = schedule
+        elif isinstance(gate, str) and params is None:
+            raise Exception("Params for the gate {} is not "
+                            "specified".format(gate))
+
```

Comment:
This is ok, for instance `add_calibration('h', [0], schedule)`

Reply:
👍 Fixed in https://github.com/Qiskit/qiskit-terra/pull/4859/commits/0e719ebb387a8187fa06d39b7c642d5185dc35cc